src/main/org/nlogo/nvm/LetBinding.java

```diff
@@ -4,9 +4,9 @@
 
 import org.nlogo.api.Let;
 
-strictfp class LetBinding {
-  final Let let;
-  Object value;
+strictfp public class LetBinding {
+  public final Let let;
+  public Object value;
```

Comment:
fwiw, I made them fields on purpose for maximum performance — this is extremely low-level, extremely inner-loop stuff.

(but did I benchmark it both ways? no. for all I know HotSpot would figure it out and the performance would be the same.)